src/entity/back-in-stock.entity.ts

```diff
@@ -14,7 +14,7 @@ export class BackInStock extends VendureEntity {
         super(input);
     }
 
-    @Column('enum', { nullable: false, enum: BackInStockSubscriptionStatus })
+    @Column('varchar')
```

Comment:
I've changed this to `varchar`, to support SQLite. We need SQLite support to be able to test with `@vendure/testing`. _(We could use other DB's in tests, but not very easily)._

In Typescript land this conversion doesn't matter, your vars will still be `BackInStockSubscriptionStatus` enums